src/web/mjs/connectors/Batoto.mjs

```diff
@@ -13,17 +13,18 @@ export default class Batoto extends AnyACG {
         this.queryMangaTitle = 'div#series-page div.title-set';
         this.queryMangaTitleText = 'h3.item-title a';
         this.queryMangaTitleFlag = 'span.item-flag';
-        this.queryMangaPages = 'nav.pager ul.pagination li.page-item:nth-last-child(2) a.page-link';
+        this.queryMangaPages = 'nav.d-none ul.pagination li.page-item:nth-last-child(2) a.page-link';
         this.queryMangas = 'div#series-list div.item-text';
         this.queryMangaLink = 'a.item-title';
         this.queryMangaFlag = 'span.item-flag';
-        this.queryChapters = 'div.chapter-list div.main a.chapt';
-        this.queryPages = /images\s*=\s*(\{.*\})\s*;/g;
+        this.queryChapters = 'div.episode-list div.main a.visited';
+        this.queryPages = /images\s*=\s*(\[.*\])\s*;/g;
     }
 
     async _getPages(chapter) {
         let request = new Request(new URL(chapter.id, this.url), this.requestOptions);
         let data = await this.fetchRegex(request, this.queryPages);
-        return Object.values(JSON.parse(data[0]));
+        data = Object.values(JSON.parse(data[0]));
+        return data.map(element => this.getAbsolutePath(`https:${element}`, request.url));
```

Comment:
Use URL constructor to build valid UVStrings with `request.url` as base (otherwise it will fail if e.g. element already starts with `https`)
`data.map(element => new URL(element, request.url))`